### PR TITLE
chore(FX-3990): update text styles on Artwork page

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves [FX-3990]

### Description
When feature flag is enabled use new styles from [Figma specs](https://www.figma.com/file/xsnfIWE7H67ZndGoEPVtN0/Alerts-2022?node-id=2103%3A30223)

### Changes
* Text size increased to `lg` for price, "For Sale", "Not For Sale"
* Text size decreased to `xs` for shipping info

### Demo
| Before  | After |
| ------------- | ------------- |
| before-1 | after-1 |
| before-2 | after-2 |
| before-3 | after-3 |
| before-4 | after-4 |
| before-5 | after-5 |

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Update some text styles on Artwork page - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-3990]: https://artsyproduct.atlassian.net/browse/FX-3990?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Artwork/Components/CommercialEditionSetInformation.tsx [hunk 1]
```diff
@@ -82,27 +84,25 @@ export class CommercialEditionSetInformation extends React.Component<Props, Stat
...
-        {!!selectedEdition! /* STRICTNESS_MIGRATION */.editionOf && (
+        {!!selectedEdition?.editionOf && (
           <>
             <Spacer mb={1} />
             <Sans size="3t" color="black30">
-              {
-                // @ts-expect-error STRICTNESS_MIGRATION --- 🚨 Unsafe legacy code 🚨 Please delete this and fix any type errors if you have time 🙏
-                selectedEdition.editionOf
-              }
+              {selectedEdition.editionOf}
```
Comment (mdole) [inline src/app/Scenes/Artwork/Components/CommercialEditionSetInformation.tsx:91]: thanks for cleaning these up!

File: src/app/Scenes/Artwork/Components/CommercialInformation.tsx [hunk 2]
```diff
@@ -148,18 +148,28 @@ const ColoredDot = styled(Box)<{ dotColor: string }>`
...
 export const SaleAvailability: React.FC<{ dotColor?: string; saleMessage: string }> = ({
   dotColor,
   saleMessage,
 }) => {
+  const enableCreateArtworkAlert = useFeatureFlag("AREnableCreateArtworkAlert")
+
+  if (enableCreateArtworkAlert) {
+    return (
+      <Flex flexWrap="nowrap" flexDirection="row" alignItems="center">
+        {!!dotColor && <ColoredDot dotColor={dotColor} />}
```
Comment (mdole) [inline src/app/Scenes/Artwork/Components/CommercialInformation.tsx:163]: Looking at the Figma, I noticed that Lois didn't include the dot in the new designs. I'm not sure if that was a mistake or if we want to get rid of it - I'll start a Slack thread with both of you and we can see what the plan is :)

**Review by mdole — DISMISSED** [review]
Great work @dimatretyak ! The code looks 💯 

Before we merge, I want to check in about whether or not we'll keep the dot for the new designs. I also wonder if we could ship these changes right away, without putting them behind the `enableCreateArtworkAlert` feature flag, just so there's less cleanup to do later. I'll start a Slack thread with Lois about both questions to clarify!

edit: thread started [here](https://artsy.slack.com/archives/C9SATFLUU/p1654072057005929)

**Review by mdole — APPROVED** [review]
nice! I think this is ready to go :)

For posterity: we're not 100% sure about the dots yet, but we're fine to release this without putting it behind the feature flag https://artsy.slack.com/archives/C9SATFLUU/p1654108171751079?thread_ts=1654072057.005929&cid=C9SATFLUU

**Review by dblandin — APPROVED** [review]
LGTM 💯

**Pull request merged**